### PR TITLE
[issue 17] create Review

### PR DESCRIPTION
**Pull request by rileyhgrant**
a 'Review' component with two variants, one to be used on a User's page, one to be used on a recipe page.

### CODE REVIEW

File: src/stories/components/Review/Review.js [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+  isUserVariant, modifierClasses, title, rating, date, name,
+  profUrl, content, recipeTitle, recipeLink,
+}) => {
+  const classes = ['review', `${modifierClasses}`].join(' ').trim();
+
+  // Kind of hacky way to turn number of stars into array to use map on
+  const stars = [];
+  for (let i = 0; i < rating; i++) {
+    stars.push({ type: 'star', key: i });
+  }
```
Comment (alecherryy) [inline src/stories/components/Review/Review.js:37]: This is fine for now, while we don't know how the rating system is implemented on the API side. However, `map` would be a cleaner way to loop through an array.

File: src/stories/components/Review/styles.scss [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+.review__recipe {
+  font-size: $font-size-sm;
+  font-family: $header-font-family;
+  font-weight: 700;
+  letter-spacing: rem(1px);
+
+  text-transform: uppercase;
+  // TODO: Change to sand-dark, I saw there was already on created
+  //   so will wait until that PR is merged then use it.
```
Comment (alecherryy) [inline src/stories/components/Review/styles.scss:20]: If this is re: the background color, this is a global style so no need to implement it.

File: src/stories/components/Review/styles.scss [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+  color: $sand;
```
Comment (alecherryy) [inline src/stories/components/Review/styles.scss:21]: This is actually orange!

File: src/stories/components/Review/styles.scss [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+}
+
+.review__title {
+  font-family: $header-font-family;
+  font-size: $font-size-lg;
+  font-weight: 700;
+}
+
+.review__sub-header {
+  width: 80%;
```
Comment (alecherryy) [inline src/stories/components/Review/styles.scss:31]: You can remove this.

File: src/stories/components/Review/styles.scss [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+  vertical-align: middle;
+  
+  a {
+    text-decoration: none;
+  }
+}
+
+.pad-left {
+  margin-left: $spacing-3;
+}
```
Comment (alecherryy) [inline src/stories/components/Review/styles.scss:65]: What is the need of this?

File: src/stories/components/Review/styles.scss [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+  font-family: $header-font-family;
+  font-size: $font-size-lg;
+  font-weight: 700;
+}
+
+.review__sub-header {
+  width: 80%;
+  font-size: $font-size-xs;
+  line-height: $line-height-loose;
+}
```
Comment (alecherryy) [inline src/stories/components/Review/styles.scss:34]: I would replace this with:
```
align-items: center;
display: flex;
font-size: $font-size-xs;
```

File: src/stories/components/Review/styles.scss [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+
+.pad-left {
+  margin-left: $spacing-3;
+}
+
+
+.review__icon {
+  &::before {
+    @include image-replace( 12px, 12px);
```
Comment (alecherryy) [inline src/stories/components/Review/styles.scss:70]: I would bump this up to `20px`.

File: src/stories/components/Review/styles.scss [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+    text-decoration: none;
+  }
+}
+
+.pad-left {
+  margin-left: $spacing-3;
+}
+
+
+.review__icon {
```
Comment (alecherryy) [inline src/stories/components/Review/styles.scss:68]: Add these styles:
```
@include image-replace( 20px, 20px);
overflow: hidden;
```

**Review by alecherryy — CHANGES_REQUESTED** [review]

File: src/stories/components/Review/styles.scss (hunk 5 above)
Comment (rileyhgrant) [reply src/stories/components/Review/styles.scss:65]: Oop good catch, absolutely nothing is the need, haha.

File: src/stories/components/Review/styles.scss (hunk 6 above)
Comment (rileyhgrant) [reply src/stories/components/Review/styles.scss:34]: Ooh nice, ```align-items: center;``` is sweeeet.

**Pull request merged**